.github/workflows/code_coverage_disable_check.py

```
@@ -0,0 +1,160 @@
+                            code_coverage_found = True
+        elif os.path.isfile(item):
+            # If it's a file, check it directly
+            if (
+                item.endswith(".tsx")
+                or item.endswith(".ts")
+                and not file_name.endswith(".test.tsx")
+                and not file_name.endswith(".test.ts")
+                and not file_name.endswith(".spec.tsx")
+                and not file_name.endswith(".spec.ts")
+            ):
+                if has_code_coverage_disable(item):
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)